README.md

```diff
@@ -104,8 +104,8 @@ Integration tests depend on having docker installed.
 
 The following environment variables can be used to avoid relying on docker.
 
-| ENV | Description |
-|--|--|
+| ENV                 | Description                   |
+|---------------------|-------------------------------|
```

Comment:
Tbh I prefer the original. It doesn't look as good, but it won't need to be maintained as much. It renders to the same look.

Reply:
Reverted.